api-reference/10 UI Components/GridBase/1 Configuration/autoNavigateToFocusedRow.md

```diff
@@ -8,4 +8,8 @@ default: true
 Automatically scrolls to the focused row when the [focusedRowKey](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/focusedRowKey.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#focusedRowKey') is changed.
```

Comment:
Can you make this more specific so that it specifies what scrolls to the focused row? For example, if you're talking about the grid, do you mean "Automatically scrolls the grid to the focused row when..."?

Reply:
It is bad practice to include a component name into short description. I can suggest the following description for both components: Automatically scrolls _the component_ to the focused row...

Reply:
Okay, I see:

```suggestion
Automatically scrolls the component to the focused row when the [focusedRowKey](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/focusedRowKey.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#focusedRowKey') is changed.
```